### PR TITLE
Fix type mismatch while taking a screenshot using the resident runner.

### PR DESCRIPTION
**Pull request by chinmaygarde**

### CODE REVIEW

**Comment by chinmaygarde** [comment]
Fixes https://github.com/flutter/flutter/issues/20550

**Review by goderbauer — APPROVED** [review]
LGTM

**Review by cbracken — APPROVED** [review]
lgtm

File: packages/flutter_tools/lib/src/ios/devices.dart [hunk 1]
```diff
@@ -376,7 +376,9 @@ class IOSDevice extends Device {
   bool get supportsScreenshot => iMobileDevice.isInstalled;
 
   @override
-  Future<Null> takeScreenshot(File outputFile) => iMobileDevice.takeScreenshot(outputFile);
+  Future<Null> takeScreenshot(File outputFile) async {
+    await iMobileDevice.takeScreenshot(outputFile);
+  }
```
Comment (Hixie) [inline packages/flutter_tools/lib/src/ios/devices.dart]: This should be a no-op change. Does it change the behaviour?

Comment (Hixie) [reply packages/flutter_tools/lib/src/ios/devices.dart]: oh never mind I see what it's doing.

**Pull request merged**